### PR TITLE
feat: Support Lottie Animation

### PR DESCRIPTION
**Pull request by dipakp2726**
# Description
Support Lottie Animations

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have followed the [Contributor Guide] when preparing my PR.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [ ] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples` or `docs`.

## Breaking Change?
- [ ] Yes, this PR is a breaking change.
- [x] No, this PR is not a breaking change.

## Related Issues

[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Conventional Commit]: https://conventionalcommits.org

### CODE REVIEW

File: packages/flame_lottie/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Flame Lottie
+
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+[![License: MIT][license_badge]][license_link]
+
+A Very Good Project created by Very Good CLI.
```
Comment (spydon) [inline packages/flame_lottie/README.md:7]: This should follow the standard for the rest of our bridge packages, not VGVs.

**Review by spydon — COMMENTED** [review]
Not very nice of you to take an issue that is assigned to someone else without discussing it with them first, because I know @tozu is working on this, so we will prioritize the PR coming from him.

**Comment by dipakp2726** [comment]
> Not very nice of you to take an issue that is assigned to someone else without discussing it with them first, because I know @tozu is working on this, so we will prioritize the PR coming from him (if they still want to do it).

I was playing with things and ended up creating this.
I wouldn't mind whether you accept this or another one😀, 
at the end of the day what we want is animation support

can you get an ETA about assignees' pr or any progress?

**Comment by spydon** [comment]
> I was playing with things and ended up creating this.
> I wouldn't mind whether you accept this or another one😀, 

👍

> can you get an ETA about assignees' pr or any progress?

I think that he's pretty close, but @tozu can probably answer that for himself. Maybe he can use your code as inspiration if he's stuck. :)

**Comment by tozu** [comment]
Hi!

Indeed I was stuck for a while, but now i seem to have found a solution. Partly because I needed to chew on this problem a bit more as well as taking inspiration from your code @dipakp2726 

I am planning to publish a PR with my preliminary solution for review in the evening 👍

**Comment by spydon** [comment]
Closing this in favor of #2157

**Pull request closed without merging**